libtelemetry/src/main/java/com/mapbox/android/telemetry/CertificateBlacklist.java

```diff
@@ -105,28 +102,30 @@ private long retrieveLastUpdateTime() {
         List<String> blacklist = obtainBlacklistContents(file);
         lastUpdateTime = Long.valueOf(blacklist.get(BLACKLIST_HEAD));
       } catch (IOException exception) {
-        logger.error(RETRIEVE_TIME_FAIL, exception.getMessage());
+        Log.e(LOG_TAG, exception.getMessage());
```

Comment:
since we already know there's a problem with `get` and `remove` on line 79 and 103. do you want to catch the exception before we release?